src/kohlrahbi/unfoldedahb/unfoldedahbtable.py

```diff
@@ -159,7 +177,8 @@ def from_ahb_table(cls, ahb_table: AhbTable, pruefi: str) -> "UnfoldedAhb":
                         segment_name=current_section_name,
                         segment_gruppe=row["Segment Gruppe"] or None,
                         segment=row["Segment"] or None,
-                        datenelement=row["Datenelement"] or None,
+                        datenelement=_split_data_element_and_segment_id(row["Datenelement"])[0],
+                        segment_id=_split_data_element_and_segment_id(row["Datenelement"])[1],
```

Comment:
ich gebe zu, das ist mittel hässlich. aber so hat pylint nicht gejammert, es gäbe zu viele locals ;)

Reply:
fair enough ;)